pulp_smash/tests/ostree/api_v2/test_sync_publish.py

```diff
@@ -39,6 +39,30 @@
 )
 
 
+def setUpModule():
+    """Skip tests if the OSTree plugin is not installed."""
+    if 'ostree' not in _get_plugin_type_ids():
+        raise unittest2.SkipTest('These tests require the OSTree plugin.')
+
+
+def _get_plugin_type_ids():
```

Comment:
This method makes use of `api.Client` and `api.json_handler`, and the `api` module uses the `utils.poll_spawned_tasks` function.